spec/imgflo-url.coffee

```diff
@@ -152,6 +156,19 @@ describe 'imgflo-url', ->
 
           expect(url).to.equal "https://imgflo.herokuapp.com/graph/#{process.env.IMGFLO_KEY}/#{process.env.IMGFLO_TOKEN_03}/passthrough.jpg?input=https%3A%2F%2Fv.cdn.vine.co%2Fr%2Fvideos%2FB5B06468B91176403722801139712_342c9a1c624.1.5.15775156368984795444.mp4.jpg%3FversionId%3DedU_LrAtIFsGvZj.Fgi0Si1bem68tBlk"
 
+      context 'with a url with uppercase JPG extension', ->
+
+        it 'should produce the correct URL', ->
+          config = getConfig()
```

Comment:
That token is generated from "key" and "secret" strings, so it won't resolve to an image, but it does test the right thing.